src/plugins/guided_onboarding/public/components/guide_panel_step.tsx

```diff
@@ -95,9 +95,9 @@ export const GuideStep = ({
             <EuiSpacer size="s" />
             <EuiText size="s" data-test-subj="guidePanelStepDescription">
               {stepConfig.descriptionList.length === 1 ? (
-                <p>{stepConfig.descriptionList[0]}</p> // If there is only one description, render it as a paragraph
+                <p css={styles.stepDescription}>{stepConfig.descriptionList[0]}</p> // If there is only one description, render it as a paragraph
```

Comment:
Can this be fixed using the `paddingSize` prop on the EuiAccordion instead? https://elastic.github.io/eui/#/layout/accordion

Reply:
Good point unfortunately using that also adds top and bottom padding, so decided to just use a custom style.